### PR TITLE
New folders

### PR DESCRIPTION
**Pull request by mbreakey3**
Creating two separate folders here. One for Test Helpers and one for the Common tests so that they can be individually uploaded to the gallery as modules. Let me know if I missed anything.

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresource.tests/43)

### CODE REVIEW

**Comment by PlagueHO** [comment]
:lgtm:

---

Reviewed 10 of 10 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-:-KH2j75SPoly1wz54dan:1417088721)*

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@{
+# Version number of this module.
+ModuleVersion = '1.10.0.0'
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: why 1.10?

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+# ID used to uniquely identify this module
+GUID = 'b3239f27-d7d3-4ae6-a5d2-d9a1c97d6ae4'
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: GUID need to be unique, use New-ModuleManifest to generate separate manifests for both modules, then fill in the details.

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) 2013 Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Module with DSC Resources for CommonTests'
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: This doesn't contain any DSC Resources, so I'd just call it something like "Common tests for DSC resources"

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+# Minimum version of the common language runtime (CLR) required by this module
+CLRVersion = '4.0'
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = @('DesiredStateConfiguration', 'DSC', 'DSCResourceKit', 'DSCResource')
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: Remove 'DSCResource' tag

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://github.com/PowerShell/xDSCResources.CommonTests/blob/master/LICENSE'
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: Provide working link

File: DSCResourceCommonTests/DSCResourceCommonTests.psd1 [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/PowerShell/xDSCResources.CommonTests'
```
Comment (KarolKaczmarek) [inline DSCResourceCommonTests/DSCResourceCommonTests.psd1]: as above

File: DSCResourcesTestsHelper/DSCResourcesTestsHelper.psd1 [hunk 7]
```diff
@@ -0,0 +1,58 @@
+@{
+# Version number of this module.
+ModuleVersion = '1.10.0.0'
```
Comment (KarolKaczmarek) [inline DSCResourcesTestsHelper/DSCResourcesTestsHelper.psd1]: all comments to the DSCResourceCommonTests.psd1 apply here.
In addition, rename this to DSCResourceTestHelper.psd1 (same for module) - let's use singular here to be consistent with the repo name

**Comment by KarolKaczmarek** [comment]
Please also leave all the files in the root as well for now, otherwise we will break all tests. We need to modify appveyor.yml files for all repos, update test templates and then we will be able to remove it as well

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 33 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHTJWzrFZ3ZVsnubZ:-KHMHTJY5thyQ3DzQWRQ:-1780143006) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L33)):*
> ```PowerShell
> 
> # Minimum version of the Windows PowerShell engine required by this module
> PowerShellVersion = '5.0'
> ```

Is there anything in particular that requires 5.0?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMHnSDnxsaY2woRcWZ:-1276780387) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
> ```PowerShell
> 
>         # A URL to the license for this module.
>         LicenseUri = 'https://github.com/PowerShell/DscResource.Tests/DSCResourceCommonTests/blob/master/LICENSE'
> ```

Could you change DSCResourceCommonTests -> DscResourceCommonTests?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 33 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHTJWzrFZ3ZVsnubZ:-KHMHm9KrxwsgrmX5uMX:691252948) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L33)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Is there anything in particular that requires 5.0?</details>
Not that I know of. What should I put instead?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 101 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHrRJIKGdmD4am6H-:-KHMHrRJIKGdmD4am6H0:-1276780387) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L101)):*
> ```PowerShell
> 
>         # A URL to the main website for this project.
>         ProjectUri = 'https://github.com/PowerShell/DSCResource.Tests/DSCResourceCommonTests'
> ```

Could you change DSCResourceCommonTests -> DscResourceCommonTests?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 33 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHTJWzrFZ3ZVsnubZ:-KHMI-JvkcO2IWAJmj9T:-1812214727) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L33)):*
<details><summary><i>Previously, mbreakey3 (Mariah) wrote…</i></summary>
> Not that I know of. What should I put instead?
> </details>
4.0 should be fine

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMI-KsFHEWgSFY0evu:32410821) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Could you change DSCResourceCommonTests -> DscResourceCommonTests?</details>
just for the url?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMI23UahsWJeM_vYla:-1330791240) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
<details><summary><i>Previously, mbreakey3 (Mariah) wrote…</i></summary>
> just for the url?</details>
also rename the folders to use "Dsc", for consistency reasons

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 12 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMIGzw3DOTDWimdms2:-KHMIGzw3DOTDWimdms3:917147104) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L12)):*
> ```PowerShell
> 
> # Script module or binary module file associated with this manifest.
> RootModule = 'DSCResource.Tests'
> ```

the root module here would be DscResourceCommonTests

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceTestsHelper/DSCResourceTestsHelper.psd1, line 12 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMILofGaFfrrb5IHCb:-KHMILofGaFfrrb5IHCc:-408179819) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceTestsHelper/DSCResourceTestsHelper.psd1#L12)):*
> ```PowerShell
> 
> # Script module or binary module file associated with this manifest.
> RootModule = 'DSCResource.Tests'
> ```

the root module here would be DscResourceTestsHelper. Btw. should we call it DscResourceTestHelper instead? (without "s" in Tests)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceTestsHelper/DSCResourceTestsHelper.psd1, line 33 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMIaC2h2LJwfeQPu6I:-KHMIaC2h2LJwfeQPu6J:1354773935) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceTestsHelper/DSCResourceTestsHelper.psd1#L33)):*
> ```PowerShell
> 
> # Minimum version of the Windows PowerShell engine required by this module
> PowerShellVersion = '5.0'
> ```

4.0 here as well

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 38 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-38:-KHMJnVvE0bxhpXsi3gf:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L38)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> as above</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 101 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHrRJIKGdmD4am6H-:-KHMJmOi__gcH1joRHhC:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L101)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Could you change DSCResourceCommonTests -> DscResourceCommonTests?</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 35 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-35:-KHMJlWcwmeZ2nQfdI7Z:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L35)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Provide working link</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMJkdcwAOMZxFOHmZm:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> also rename the folders to use "Dsc", for consistency reasons
> </details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 32 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-32:-KHMJjeTN_IYAaM_ChoZ:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L32)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Remove 'DSCResource' tag</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 33 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHTJWzrFZ3ZVsnubZ:-KHMJt4JWOHXO3kGxU2R:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L33)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> 4.0 should be fine</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 18 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-18:-KHMJuox-eJVwTs-ThgL:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L18)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> This doesn't contain any DSC Resources, so I'd just call it something like "Common tests for DSC resources"</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 6 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-6:-KHMJwJb_FdTOqfPaFsF:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L6)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> GUID need to be unique, use New-ModuleManifest to generate separate manifests for both modules, then fill in the details.</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 3 \[r2\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KH2QOZsEPcg2edrKUge-r2-3:-KHMJyIuzU3locbxWatj:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/c5c55262c1831bbb2b2af6cb4e016f0b047dd180/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L3)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> why 1.10?</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 12 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMIGzw3DOTDWimdms2:-KHMJzg8nXAmd56zq3Jb:-499146009) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L12)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> the root module here would be DscResourceCommonTests</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMMwHkpdezKYbP0Y9B:559731981) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
<details><summary><i>Previously, mbreakey3 (Mariah) wrote…</i></summary>
> Done.</details>
this one (DSCResourceCommonTests) still needs to be renamed - both folder and files inside

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by mbreakey3** [comment]
*[DSCResourceCommonTests/DSCResourceCommonTests.psd1, line 98 \[r3\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-KHMHnSDnxsaY2woRcWY:-KHMNPLW__b39vYDXikk:-2025109262) ([raw file](https://github.com/powershell/dscresource.tests/blob/23e5b345d1301a76f69d99cf7cafa80071a2b61b/DSCResourceCommonTests/DSCResourceCommonTests.psd1#L98)):*
<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> this one (DSCResourceCommonTests) still needs to be renamed - both folder and files inside
> </details>
Oops, forgot to re-add that folder

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by KarolKaczmarek** [comment]
:lgtm:

<details><summary><i>Previously, KarolKaczmarek wrote…</i></summary>
> Please also leave all the files in the root as well for now, otherwise we will break all tests. We need to modify appveyor.yml files for all repos, update test templates and then we will be able to remove it as well</details>


---







*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43#-:-KHMNfpyByYydQ9JkiTJ:980077401)*

**Comment by KarolKaczmarek** [comment]
Reviewed 7 of 16 files at r1, 4 of 17 files at r3, 9 of 9 files at r4.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/43)*

**Comment by PlagueHO** [comment]
@KarolKaczmarek, @mbreakey3: Not completely related, but do you know why Reviewable is outputting each comment etc into a new message? It is supposed to combine all comments you make in a single review session into a single message. I only ask because I just got an e-mail storm from this PR :smile:Are you clicking Publish after each comment?

**Comment by KarolKaczmarek** [comment]
no, I clicked publish only once and it produced this comment:
```
Reviewed 7 of 16 files at r1, 4 of 17 files at r3, 9 of 9 files at r4.
Review status: all files reviewed at latest revision, 3 unresolved discussions.
```

Afair it was always working like this, at least for me. Can you point to the example PR where reviewable produced a summary of all comments? Perhaps there's some setting for it?

**Comment by PlagueHO** [comment]
@KarolKaczmarek - I was more used to seeing the comments like this:
https://github.com/PowerShell/xWebAdministration/pull/98#issuecomment-209717207
https://github.com/PowerShell/xActiveDirectory/pull/86#issuecomment-208817066
https://github.com/PowerShell/xWebAdministration/pull/98#issuecomment-203165137


Perhaps the Reviewable people changed something? It really isn't a big deal, I just wondered what happened as I did quite like having everything joined together in a single comment.

**Comment by KarolKaczmarek** [comment]
yes, I definitely like it more as well. Not sure why it added every comment separately in this PR... @TravisEz13 ?

**Pull request merged**